chapter2/tests/t16/project.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+id = "io.buildpacks.php-slow-script"
+name = "buildpacks - slow php reporting page"
+version = "1.0.0"
+BP_PHP_VERSION="8.1.7"
```

Comment:
incorrect, you should define it as ENV var
[BP_PHP_VERSION](https://paketo.io/docs/howto/php/)

https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md#iobuildpacksbuildenv-optional

Reply:
> incorrect, you should define it as ENV var [BP_PHP_VERSION](https://paketo.io/docs/howto/php/)

@dmitry-mightydevops 
Done